modules/appeals_api/spec/support/shared_examples_for_model_validations.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+shared_examples 'shared model validations' do |opts|
+  opts = opts.with_indifferent_access
+
+  # All claimant headers across appeal models
+  all_claimant_headers = %w[X-VA-Claimant-First-Name X-VA-Claimant-Middle-Initial X-VA-Claimant-Last-Name
+                            X-VA-Claimant-Birth-Date].freeze
+
+  describe '#birth_date_is_in_the_past' do
+    next unless opts[:validations].include? :birth_date_is_in_the_past
+
+    before { appeal.auth_headers = appeal.auth_headers.merge 'X-VA-Birth-Date' => '3000-01-02' }
+
+    it 'errors with source at the veteran birth date header' do
```

Comment:
I wonder if we could be more descriptive, since the describe references the validation method name, and then we jump right to 'errors with...' Maybe something like 'errors with...when birth date is in the future', or maybe use context to pull it out? 

I'll get this merged and make a quick PR with options that we can discuss tomorrow.